stdlib/2/UserString.pyi

```diff
@@ -13,9 +13,9 @@ class UserString(Sequence[UserString]):
     def __hash__(self) -> int: ...
     def __len__(self) -> int: ...
     @overload
-    def __getitem__(self: _UST, i: int) -> _UST: ...
+    def __getitem__(self: _UST, i: int): ...  # type: ignore
     @overload
-    def __getitem__(self: _UST, s: slice) -> _UST: ...
+    def __getitem__(self: _UST, s: slice): ...  # type: ignore
```

Comment:
Ah, yes, I remember now, to avoid this error:

```
stdlib/2/UserString.pyi:64: error: Definition of "__getitem__" in base class "UserString" is incompatible with definition in base class "MutableSequence"
```

Reply:
This should be fixable by annotating `MutableString.__getitem__()` with `# type: ignore` as well, instead of removing the return value.